Footer styling

joshuazhu17 opened the pull request:
<!--
    Tag your PR title with the components that it touches along with
    the type of change (feat, fix, refactor)
    E.g. "fix: fix activity 1 button placement" or "feat: add favicon and make footer thinner"
-->

## Summary

Closes #<!-- issue number -->

<!-- Enumerate changes you made and why you made them -->
<!-- list any new dependencies required for this change -->

## Test Plan

<!--
    How did you manually test your change? How do you know that your code works?
    Add supporting screenshots of the feature in play, terminal pastes, etc. as necessary
-->

joshuazhu17 committed:
Footer styling

joshuazhu17 committed:
Added dotted line

bliutech reviewed (changes requested):
Great start. Here are a few changes.

<img width="1324" alt="image" src="https://user-images.githubusercontent.com/67720812/233139530-4db196e8-e365-4e76-ae4a-38c986b969a4.png">

Could you make these horizontally spread instead of vertically spread? Maybe use SVGs instead of just text similar to the LA CTF website (https://lactf.uclaacm.com/)?

Additionally, could you add the link to our CTF Time (https://ctftime.org/team/186494) and the main ACM Cyber website (https://acmcyber.com/)?

Nice work so far.

Arc-blroth committed:
Merge branch 'main' into footer-styling

Arc-blroth committed:
fix nav (FINAL).patch

apparently all browsers just straight up ignore font-size for the
height of a div when the font is monospace

AAAAAAAAAAAAAAA

see https://stackoverflow.com/a/39870273

Arc-blroth committed:
i don't think this is possible

Arc-blroth committed:
"good enough"

Arc-blroth committed:
the dark side

bliutech committed:
Merge branch 'main' into footer-styling

bliutech reviewed (approved):
LGTM.
